service/storage_proxy.cc

```diff
@@ -2785,6 +2788,14 @@ void storage_proxy_stats::stats::register_stats() {
         sm::make_total_operations("cas_dropped_prune", cas_replica_dropped_prune,
                        sm::description("how many times a coordinator did not perform prune after cas"),
                        {storage_proxy_stats::current_scheduling_group_label()}).set_skip_when_empty(),
+
+        sm::make_total_operations("received_hint_count", received_hint_count,
```

Comment:
we have deprecated collectd, use counter or gauge.
and a better name will be received_hints_total